src/olympia/stats/management/commands/index_stats.py

```diff
@@ -34,7 +34,7 @@
 
 def gather_index_stats_tasks(index, addons=None, dates=None):
     """
-    Return the list of task groups to execute to index statistics for the given
+    Return list of tasks to execute to index statistics for the given
```

Comment:
This change is also crucial to avoid unnecessary nesting. 

Returning a list of groups didn't get us anything besides added complexity later. So now this function unwraps the group created by each `create_chunked_tasks_signatures` call, returns a flat list of tasks, and that list is _then_ fed into a `group()` either by `gather_index_stats_tasks()` (used in `reindex`) or the command `handle()` method.